### ISSUE TITLE
Incompatibility with Erlang 24

### ISSUE THREAD

**Issue by vkatsuba** [rabbitmq/looking-glass#18]
During an attempt to add Erlang OTP 24.0 to the GitHub Actions the CT was crashed - https://github.com/vkatsuba/looking_glass/runs/2570008061
```erlang
%%% lg_SUITE ==> all.callgrind_running: FAILED
%%% lg_SUITE ==> {'EXIT',{badarg,{erlang,integer_to_list,
                        [{59,1}],
                        [{error_info,#{module => erl_erts_errors}}]}}}

=ERROR REPORT==== 12-May-2021::20:18:37.985067 ===
Error in process <0.487.0> with exit value:
{badarg,[{erlang,integer_to_list,
                 [{97,1}],
                 [{error_info,#{module => erl_erts_errors}}]},
         {lg_callgrind,write_call,3,
                       [{file,"/__w/looking_glass/looking_glass/src/lg_callgrind.erl"},
                        {line,414}]},
         {lg_callgrind,'-write_mfas/3-lc$^0/1-0-',3,
                       [{file,"/__w/looking_glass/looking_glass/src/lg_callgrind.erl"},
                        {line,388}]},
         {lg_callgrind,write_mfas,3,
                       [{file,"/__w/looking_glass/looking_glass/src/lg_callgrind.erl"},
                        {line,388}]},
         {lg_callgrind,handle_return_to,5,
                       [{file,"/__w/looking_glass/looking_glass/src/lg_callgrind.erl"},
                        {line,264}]},
         {lg_file_reader,fold1,3,
                         [{file,"/__w/looking_glass/looking_glass/src/lg_file_reader.erl"},
                          {line,44}]},
         {lg_file_reader,fold,3,
                         [{file,"/__w/looking_glass/looking_glass/src/lg_file_reader.erl"},
                          {line,37}]},
         {lg_callgrind,profile,3,
                       [{file,"/__w/looking_glass/looking_glass/src/lg_callgrind.erl"},
                        {line,96}]}]}
```

**Comment by michaelklishin** [comment]
Erlang 24 is less than 24 hours old.

From the looks of it, it must be the new `line:position` metadata that [`lg_callgrind` does not expect](https://github.com/rabbitmq/looking_glass/blob/master/src/lg_callgrind.erl#L391-L416) to encounter.
Feel free to submit a pull request, this is open source software after all.

**Comment by vkatsuba** [comment]
Fixed in [a7958cb](https://github.com/rabbitmq/looking_glass/commit/a7958cbecdb63f6e03ebf22e4328921edc09780a).
